pkg/services/ngalert/schedule/schedule_unit_test.go

```diff
@@ -59,7 +59,7 @@ func TestProcessTicks(t *testing.T) {
 	mockedClock := clock.NewMock()
 
 	notifier := &AlertsSenderMock{}
-	notifier.EXPECT().Send(mock.Anything, mock.Anything).Return()
+	notifier.EXPECT().Send(mock.Anything, mock.Anything, mock.Anything).Return()
```

Comment:
Changes in this file are:
- Adding a first argument corresponding to the `context.Context`
- Checking that the third (not second) argument is of type `definitions.PostableAlerts`